### ISSUE TITLE
Proposal: RestrictedDiet :: LowCarbDiet

### ISSUE THREAD

**Issue by tomkenkel** [schemaorg/schemaorg#3176]
Thing > Intangible > Enumeration > RestrictedDiet :: LowCarbDiet

A diet focused on reduced carbohydrate intake.

A member value for enumeration type: [RestrictedDiet](https://schema.org/RestrictedDiet)

**Comment by chrisspradling1980** [comment]
Crystalline 
-please take note of the word not tokenized
Gel: resulting as substance pertaining to a diet calling for RestrictedDiet
I agree with the inverse Domain requests particular to the EnvironmentType self incrimination occurs because of USCA titles and federal Scheduling for possession. Symbolic expression is needed to timecode token experiences. This will never stop with humans.

**Comment by RichardWallis** [comment]
See also #3211

**Comment by GinMorgWireless** [comment]
#3211 offers more information than total carbs. People who need the tags LowSucrose and LowMaltose need the lower resolution than just carb levels. They often need to eat glucose and fructose to make up for the reduced sucrose, maltose and starch. The LowCarb tag is not suitable on its own as foods can be "low carb" and still contain too much sucrose, maltose or starch for safe digestion in this population.

**Comment by fthobe** [comment]
@GinMorgWireless wouldn't that be different kinds of diets?

**Comment by GinMorgWireless** [comment]
Yeah I’d love to get these tags added as low carb and low disaccharide
tags, even low monosaccharides like fructose would be helpful for lots of
people. It would actually make a huge difference for people who are looking
for these specific medical dietary needs.

I’m not sure how to progress something like this further through the schema
though??

**Comment by fthobe** [comment]
Can you make a complete list inside a table with: 

- name of the diet
- comment describing the diet

You can find the full list of currently supported diets here: 
[RestrictedDiet](https://schema.org/RestrictedDiet)

And if you click on any of the elements in the list you can find the comment here: 
<img width="657" alt="Image" src="https://github.com/user-attachments/assets/b779dc5f-532e-46e8-888f-60380c593364" />

If you provide me with a name and comment I will check if I get interest on the formal PR to add it to schema.org.

**Comment by GinMorgWireless** [comment]
Carbohydrate Malabsorption Special Medical Diets

LowLactoseDiet
A diet appropriate for people with lactose intolerance.
LowSucroseDiet A diet appropriate for people with sucrose intolerance. Some
people with CSID only need to restrict sucrose
LowMaltoseDiet A diet appropriate for people with maltose intolerance. Some
people with CSID must also restrict maltose
LowStarchDiet A diet appropriate for people with starch intolerance. Some
people with CSID must also restrict starch
SIDDiet A diet appropriate for people with sucrase-isomaltase deficiency
(Also known as CSID). Restricts Sucrose, Maltose, Isomaltose & Starch.
LowTrehaloseDiet A diet appropriate for people with trehalose
intolerance. People
who cant digest mushroom sugar
LowFructoseDiet A diet appropriate for people with fructose intolerance. People
who do not digest fructose
FructoseFreeDiet A diet appropriate for people with Hereditary Fructose
Intolerance (HFI). People who digest fructose and it damages their body

**Comment by TechHappyNZ** [comment]
https://github.com/schemaorg/schemaorg/issues/3211#issuecomment-2811614022 
#3211

**Comment by fthobe** [comment]
> Some people with CSID only need to restrict sucrose LowMaltoseDiet

Please do not add medical advise to the comment. 

With the markdown table generator you can properly format the table for usage on GitHub:  https://www.tablesgenerator.com/markdown_tables

**Comment by alex-jansen** [comment]
Completed by alternative implementation in #4667

**Comment by TechHappyNZ** [comment]
How do I create a new proposal that would add enumeration types for the other suggested diets in this issue? Does each one need a separate issue created??

**Comment by fthobe** [comment]
Hey @TechHappyNZ it was decided to not extend the definitions present, with the new release you will be able to implement your own diets within the code your page to have more flexibility.
